### ISSUE TITLE
Controlling when to open the popover

### ISSUE THREAD

**Issue by arlen-yu** [TeamWertarbyte/material-ui-chip-input#119]
I want to be able to open and close the popover with the click of a button, but it doesn't seem to be working. Right now I'm trying to modify the open prop of AutoComplete:

```jsx
<ChipInput
onFocus={() => this.onFocus()}
onBlur={() => this.onBlur()}
dataSource={this.props.dataSource}
value={this.state.chips}
onRequestAdd={(chip) => this.handleRequestAdd(chip)}
onRequestDelete={(deletedChip) => this.handleRequestDelete(deletedChip)}
underlineShow={false}
style={{marginBottom: '-21px'}}
chipRenderer={this.chipRenderer}
open={this.state.open}
allowDuplicates
fullWidth
/>
```

And here's the button: 
```jsx
<IconButton onTouchTap={() => this.openPopover()}>
  <DropdownArrow />
</IconButton>
```

where `openPopover()` simply modifies the state to have `open: true`. Any solutions?

**Comment by leMaik** [comment]
The underlying `AutoComplete` component only opens the popup menu when the `open` prop _changes_ to `true`. So you need to add `onClose={() => this.setState({ open: false })` to your `ChipInput`.

**Comment by arlen-yu** [comment]
Thanks for the response. I've implemented the changes and checked that the state is changing from true to false yet still the popover isn't opening (or closing, or changing at all). Not sure what the reason for this is, perhaps the prop isn't properly passed down to the AutoComplete? 

Is there any other workaround solutions you can think of?

**Comment by leMaik** [comment]
@arlen-yu I made sure that the `open` prop is passed down (and not being able to open the popover when setting `open={false}` also confirms this).

I'll try to throw together a minimal example, stay tuned… :+1:

**Comment by leMaik** [comment]
The following example works fine:

```jsx
import React from 'react'
import ChipInput from 'material-ui-chip-input'

export default class ControlledOpenClose extends React.Component {
  constructor (props) {
    super(props)
    this.state = {
      open: false
    }
  }

  handleToggleOpen = () => {
    this.setState({ open: !this.state.open })
  }

  render () {
    return (
      <div>
        <button onClick={this.handleToggleOpen}>{this.state.open ? 'close' : 'open'} auto complete popover</button>
        <ChipInput
          {...this.props}
          open={this.state.open}
          onClose={() => this.setState({ open: false })}
          fullWidth
        />
      </div>
    )
  }
}
```

**Comment by arlen-yu** [comment]
I replicated your example exactly and it doesn't work on my end... Very weird!

**Comment by arlen-yu** [comment]
Updated my material ui to 0.18.0 and it still doesn't seem to work. What versions are you using?

**Comment by arlen-yu** [comment]
Ah, they fixed the open prop of Autocomplete in 0.18.1 according to the change log. Thanks for your help.
